modules/outconAdapter.md

```diff
@@ -0,0 +1,24 @@
+# Overview
+
+```
+Module Name: outconAdapter
+Module Type: Bidder Adapter
+Maintainer: mfolmer@dokkogroup.com.ar
+```
+
+# Description
+
+Module that connects to Outcon demand sources
+
+# Test Parameters
+```
+    var adUnits = [
+        {
+        bidder: 'outcom',
+        params: {
+          internalId: '12345678',
+          publisher: '5d5d66f2306ea4114a37c7c2',
+          demo: true
+        }
```

Comment:
Two points of follow-up feedback:
- there is a missing `}` for this overall object block.
- the `bidder` value is using `outcom` instead of `outcon`

Can you update these items so it's properly formatted and consistent with your adapter?